### DISCUSSION TITLE
Adding Style in PropertyColumn also changes the style of the header.

### DISCUSSION THREAD

**Discussion by thieng101** [microsoft/fluentui#33497] [Q&A]
I am using FluentDataGrid and PropertyColumn. When I add Style (color: red) in PropertyColumn, it also applies to the header which I don't want. I did the same thing in the demo website which is :

`<PropertyColumn Property="@(c => c!.State)" Style="color: #af5f00 ;" />`

I don't know where is the problem. Please help. Thank you.

**Comment by ValentinaKozlova** [comment]
Could you please share a link on stackblitz with your example?
